newIDE/app/src/Profile/Subscription/SubscriptionSuggestionContext.js

```diff
@@ -0,0 +1,47 @@
+// @flow
+import * as React from 'react';
+import SubscriptionDialog from './SubscriptionDialog';
+import { type SubscriptionDialogDisplayReason } from '../../Utils/Analytics/EventSender';
+
+type SubscriptionSuggestionState = {|
+  openSubscriptionDialog: ({|
+    reason: SubscriptionDialogDisplayReason,
+    preStep?: 'subscriptionChecker',
```

Comment:
what is this prestep for?

Reply:
To have more info on the context: If the subscription dialog has been opened from the SubscriptionChecker dialog.
I don't know if `preStep` is a great naming though.
And I don't know if this piece of info is useful but I thought it might help at one point